docs/operation/lerobot_guide.rst

```diff
@@ -0,0 +1,306 @@
+==========================
+LeRobot X Aloha User Guide
+==========================
+
+Virtual Environment Setup
+=========================
+
+Containerization is crucial for running machine learning models to avoid dependency conflicts. You can either use a Virtual Environment (venv) or Conda for this purpose.
+
+Using Virtual Environment (venv)
+--------------------------------
+
+1. Install the virtual environment package:
+
+   .. code-block:: bash
+
+      sudo apt-get install python3-venv
+
+2. Create a virtual environment:
+
+   .. code-block:: bash
+
+      python3 -m venv ~/lerobot  # Creates a venv "lerobot" in the home directory
+
+3. Activate the virtual environment:
+
+   .. code-block:: bash
+
+      source ~/lerobot/bin/activate
+
+Using Conda
+-----------
+
+1. Create a virtual environment:
+
+   .. code-block:: bash
+
+      conda create -n lerobot python=3.10
+
+2. Activate the virtual environment:
+
+   .. code-block:: bash
+
+      conda activate lerobot
+
+.. note::
+   Use either `venv` or `Conda` based on your preference, but **do not** mix them to avoid dependency issues.
+
+Clone Repository
+================
+
+For users of Aloha Stationary:
+
+1. Clone the LeRobot repository:
+
+   .. code-block:: bash
+
+      cd ~
+      git clone https://github.com/huggingface/lerobot.git
```

Comment:
Do we want to tag a specific version of lerobot that we know works correctly right now? Something like 

```
$ git clone -b f431a08efa74a9538ec719647745fa658faa85c3 https://github.com/huggingface/lerobot.git
```